packages/@react-spectrum/actionbar/docs/ActionBar.mdx

```diff
@@ -0,0 +1,276 @@
+        {item => (
+          <Item key={item.key}>
+            {item.label}
+          </Item>
+        )}
+      </ActionBar>
+    </ActionBarContainer>
+  );
+}
+
+<DynamicActionBar />
+```
+
+### Layout
+ActionBarContainer accepts an ActionBar alongside any renderable children. It accepts the typical selection of [style props](#props), allowing you to customize its layout.
+The example below applies a static height on the ActionBarContainer instead of matching the height of the TableView.
+
+```tsx example
+import {Cell, Column, Row, TableBody, TableHeader, TableView} from '@react-spectrum/table';
+import {Text} from '@react-spectrum/text';
+
+function ActionBarTable(props) {
+  // Same bar items as example above...
+  ///- begin collapse -///
+  let barItems = [
+    {key: 'edit', label: 'Edit'},
+    {key: 'copy', label: 'Copy'},
+    {key: 'delete', label: 'Delete'}
+  ];
+  ///- end collapse -///
+  let columns = [
+    {name: 'Name', uid: 'name'},
+    {name: 'Type', uid: 'type'},
+    {name: 'Level', uid: 'level'}
+  ];
+
+  let rows = [
+    {id: 1, name: 'Charizard', type: 'Fire, Flying', level: '67'},
+    {id: 2, name: 'Blastoise', type: 'Water', level: '56'},
+    {id: 3, name: 'Venusaur', type: 'Grass, Poison', level: '83'},
+    {id: 4, name: 'Pikachu', type: 'Electric', level: '100'}
+  ];
+
+  let [selectedKeys, setSelectedKeys] = React.useState(new Set([2]));
+  return (
+    <ActionBarContainer height="size-5000">
+      <TableView aria-label="Table with controlled selection" selectionMode="multiple" selectedKeys={selectedKeys} onSelectionChange={setSelectedKeys} height="size-3000">
+        <TableHeader columns={columns}>
+          {column => (
+            <Column key={column.uid} align={column.uid === 'level' ? 'end' : 'start'}>
+              {column.name}
+            </Column>
+          )}
+        </TableHeader>
+        <TableBody items={rows}>
+          {item => (
+            <Row>
+              {columnKey => <Cell>{item[columnKey]}</Cell>}
+            </Row>
+          )}
+        </TableBody>
+      </TableView>
+      <ActionBar
+        items={barItems}
+        isEmphasized
+        selectedItemCount={selectedKeys === 'all' ? selectedKeys : selectedKeys.size}>
+        {item => (
+          <Item key={item.key}>
+            {item.label}
+          </Item>
+        )}
+      </ActionBar>
+    </ActionBarContainer>
+  );
+}
+```
+
+### Accessibility
+The contents of the ActionBar should follow the same accessibility guidelines as ActionGroup's items, see the [ActionGroup docs](./ActionGroup.html#accessibility) for more details.
+
+### Internationalization
+To internationalize an ActionBar, the content provided to all child items should be localized. A localized string for the selection count and clear button `aria-label` will be provided
+automatically. For languages that are read right-to-left (e.g. Hebrew and Arabic), the layout of the ActionBar is automatically flipped.
+
+## Events
```

Comment:
I think we need to put a height on the table? the usual use case is that the table shrinks upwards to make room for the actionbar instead of the page shifting down to make room for it